### DISCUSSION TITLE
Ore-Control in a World Painter world

### DISCUSSION THREAD

**Discussion by BenjiEnergy** [DerFrZocker/Ore-Control#19] [Q&A]
Is it possible to fully edit the values, completely remove ores from some biomes, and amplify the amount in others, if using a World Painted map, or is that an already generated map?

**Comment by DerFrZocker** [comment]
Ore Control can change the ore generation of World Painter map if you enable "allow Minecraft to populate the entire terrain" in World Painter during export.
